documentation/book/proc-scaling-up-a-kafka-cluster.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// ???
+
+[id='proc-scaling-up-a-kafka-cluster-{context}']
+
+= Scaling up a Kafka cluster
+
+This procedure describes how to increase the number of brokers in a Kafka cluster.
+
+.Prerequisites
+
+* An existing Kafka cluster.
+* A _reassignment JSON file_, `_reassignment.json_`, of how partitions should be reassigned to brokers in the enlarged cluster.
```

Comment:
A _reassignment JSON file_ named `_reassignment.json_` that describes how partitions should be reassigned to brokers in the enlarged cluster.